Dockerfile

```diff
@@ -146,7 +158,7 @@ RUN set -ex\
 
 RUN rm -Rf node_modules config_app/node_modules
 
-EXPOSE 8080 8443 7443 9091 55443
+EXPOSE 8080 8443 7443 9000 9091 55443
```

Comment:
Why expose port 9000, new UI should be routed via the same port, there could be a header or a cookie switch between new and old ui

Reply:
@syed How would the header or cookie be set? If a user is trying to navigate to the old ui at `/` and the new ui at `/`, how would we set the header?

Reply:
and in the new UI we can have another banner saying to go back to the old UI click here. We could have this till we have feature parity. Then we just remove the old UI. For this to work, both the old UI and new UI should run on the same port

Reply:
Ok, I like that. It'll also mean we don't need to change anything related to `cors_origin`